proto/asn1-pdu/asn1_universal_types_to_der.cc

```diff
@@ -67,6 +67,39 @@ void Encode(const BitString& bit_string, std::vector<uint8_t>& der) {
   }
 }
 
+void Encode(const ObjectIdentifier& object_identifier,
+            std::vector<uint8_t>& der) {
+  // Save the current size in |tag_len_pos| to place tag and length
+  // after the value is encoded.
+  const size_t tag_len_pos = der.size();
+
+  uint8_t root = object_identifier.root();
+  uint8_t small_identifier = object_identifier.small_identifier();
+  auto subidentifier = object_identifier.subidentifier();
```

Comment:
Note: This does force an explicit copy, rather than using it by reference. However, that's OK, for the reasons mentioned below: